client/app/veeam-enterprise/dashboard/license/veeam-enterprise-license.controller.js

```diff
@@ -0,0 +1,44 @@
+class VeeamEnterpriseLicenseCtrl {
+  constructor($uibModalInstance, action, serviceName, VeeamEnterpriseService) {
+    this.$uibModalInstance = $uibModalInstance;
+    this.action = action;
+    this.serviceName = serviceName;
+    this.VeeamEnterpriseService = VeeamEnterpriseService;
+
+    this.form = {};
+  }
+
+  dismissModal() {
+    this.$uibModalInstance.dismiss('cancel');
+  }
+
+  submitForm(form) {
+    if (form.$valid) {
```

Comment:
Can't you prevent pushing the button instead ?

Reply:
It comes from ngSubmit